pkgs/by-name/tu/turbo-unwrapped/package.nix

```diff
@@ -15,19 +15,19 @@
   zlib,
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "turbo-unwrapped";
-  version = "2.4.2";
+  version = "2.4.5-canary.4";
 
   src = fetchFromGitHub {
     owner = "vercel";
     repo = "turbo";
```

Comment:
the repo has been renamed to `turborepo`. Github is smart enough to do a redirect, but it might make sense to point directly to the right repo here.